backend/src/main/java/com/agh/introwertycznelosie/config/ExamController.java

```diff
@@ -43,23 +46,27 @@
     public Long postNewMajor(@RequestBody ExamMockup examMockup) {
```

Comment:
To się tak powinno nazywać? New Major?

Reply:
Tu pewnie był kopiuj wklej, to przy okazji poprawię